mcsema/BC/Function.cpp

```diff
@@ -260,6 +288,132 @@ static llvm::BasicBlock *GetOrCreateBlock(TranslationContext &ctx,
   return block;
 }
 
+static void LiftExceptionFrameLP(TranslationContext &ctx,
```

Comment:
This is possibly a stupid question... but.. can any of the code here that manually write LLVM IR be replaced with a .ll / .bc file that is loaded and modified? It'd be kind of like going from old mcsema semantics to remill semantics.

If its possible, it'd be easier to maintain and would result in less code.